insanely_fast_whisper_rocm/core/segmentation.py

```
@@ -0,0 +1,1161 @@
+        "consequently",
+        "meanwhile",
+        "otherwise",
+        "instead",
+        "besides",
+        "additionally",
+        "similarly",
+        "likewise",
+        "further",
+        "also",
+        "plus",
+        "moreover",
```

Ruff (0.14.7)

Sets should not contain duplicate item `"moreover"`

Remove duplicate item

(B033)